source/core/Ky.ts

```diff
@@ -0,0 +1,322 @@
+/* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
```

Comment:
Don't disable it for the whole file. Disable it where it needs to be, but keep in mind that most cases, the rule is correct.